NDTensors/test/blocksparse.jl

```diff
@@ -226,12 +234,16 @@ using GPUArraysCore: @allowscalar
   end
 
   @testset "svd on $dev" for dev in devs
+    if dev == NDTensors.mtl
+      NDTensors.default_eltype() = Float32
+    end
+    atol = 10^(0.75 * log10(eps(real(elt))))
```

Comment:
It seems like there is a lot of logic in:
https://github.com/ITensor/ITensors.jl/blob/408516dce2e0182e04a6fb27b501b04a31b630fc/NDTensors/test/ITensors/TestITensorDMRG/TestITensorDMRG.jl
that is helpful in other tests. Probably best to make a module `TestNDTensorsUtils` in `NDTensors/test/TestNDTensorsUtils/TestNDTensorsUtils.jl` that defines some general helper functions for tests.

That could be merged with `NDTensors/test/devices_list.jl`, probably that should go in `TestNDTensorsUtils` as well.